tests/ml_tools/test_qnn.py

```diff
@@ -347,9 +347,46 @@ def test_variational_transformed_module(
     assert torch.allclose(tm_pred, (output_range[0] * pred) + output_range[1])
 
 
-def test_config_qnn_input_transform() -> None:
+def test_config_empty_fm() -> None:
     fm_config = FeatureMapConfig()
```

Comment:
If you default it, I don't think this would be necessary anymore.

Reply:
Opps! True. Missed the point of the test altogether 😅